### PR TITLE
Paint-195: Espresso tests for LayerIntegrationTest

### PR DESCRIPTION
**Pull request by AlexWieland**
Programmed Layer Tests, fixed Pipette Bug, refactored Tests

### CODE REVIEW

**Comment by AlexWieland** [comment]
Deleted old LayerIntegrationTest - class

**Comment by AlexWieland** [comment]
All Tests are fixed, deleted old Layerintegration-Testclass

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/LayerIntegrationTest.java [hunk 1]
```diff
@@ -163,6 +212,312 @@ public void testUndoRedoLayerAdd() {
...
+		onLayerMenuView().performOpen();
+		onLayerMenuView().performAddLayer();
+		onLayerMenuView().performClose();
+		onView(isRoot()).perform(touchAt(screenPointRight));
+
+		ToolBarViewInteraction.onToolBarView().performSelectTool(ToolType.TRANSFORM);
+		try {
+			Thread.sleep(100);
+		} catch (Exception e) {
+			e.printStackTrace();
```
Comment (ThomasSchwengler) [inline Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/LayerIntegrationTest.java]: please change to `Log.e(e.getMessage())`

**Comment by AlexWieland** [comment]
Test this

**Comment by ThomasSchwengler** [comment]
test this please

**Pull request merged**